src/main/scala/com/sksamuel/scapegoat/inspections/collections/PredefSeqIsMutable.scala

```diff
@@ -11,24 +11,22 @@ class PredefSeqIsMutable
       explanation = "Predef.Seq aliases scala.collection.mutable.Seq. Did you intend to use an immutable Seq?"
     ) {
 
-  def inspector(context: InspectionContext): Inspector = new Inspector(context) {
-    override def postTyperTraverser =
-      if (isScala213) None
-      else
-        Some(
-          new context.Traverser {
+  override def isEnabled = !isScala213
 
-            import context.global._
+  def inspector(context: InspectionContext): Inspector = new Inspector(context) {
+    override def postTyperTraverser = new context.Traverser {
+      import context.global._
 
-            override def inspect(tree: Tree): Unit = {
-              tree match {
-                case DefDef(_, _, _, _, _, _) if tree.symbol.isAccessor =>
-                case TypeTree() if tree.tpe.erasure.toString() == "Seq[Any]" =>
-                  context.warn(tree.pos, self)
-                case _ => continue(tree)
-              }
-            }
+      override def inspect(tree: Tree): Unit = {
+        if (!isScala213) {
```

Comment:
Can you use `isEnabled` here instead? I guess having this logic in one place makes it easier to maintain in case this needs to change.

Reply:
Ah, thanks for spotting. That is actually to be removed. The `isEnabled` check is done at the framework layer (https://github.com/sksamuel/scapegoat/blob/256493c229cf19fe802333ac6215e00b524d717b/src/main/scala/com/sksamuel/scapegoat/plugin.scala#L253).
Let me fix that in a separate PR